enclave_wrangler/objects_api.py

```diff
@@ -410,15 +410,20 @@ def find_and_add_missing_csets_to_db(con: Connection = None, schema=SCHEMA):
 
 
 def fetch_cset_and_member_objects(
-    since: Union[datetime, str] = '', codeset_ids: List[int] = [], handle_issues=True, verbose=False
+    since: Union[datetime, str] = '', codeset_ids: List[int] = [], flag_issues=True, verbose=False,
+    try_fixing_issues_immediately=False
```

Comment:
## `try_fixing_issues_immediately=False`
Now, by default, no longer tries to fix mid-refresh; fixes at end of refresh, as it takes members time to appear.